components/GymPlan/WorkoutSet.js

```diff
@@ -0,0 +1,86 @@
+import styled from "styled-components";
+
+export default function WorkoutSet({ setName }) {
+  return (
+    <>
+      <StyledSection>
```

Comment:
We are having a formular here. Please check if you are meeting criteria for formulars ;) 
(form, inputs, label, submitbuttton)